src/smartHandler.ts

```diff
@@ -295,8 +314,10 @@ export class SMARTHandler implements Authorization {
     async getAllowedResourceTypesForOperation(request: AllowedResourceTypesForOperationRequest): Promise<string[]> {
         let allowedResources: string[] = [];
         const allResourceTypes: string[] = this.fhirVersion === '4.0.1' ? BASE_R4_RESOURCES : BASE_STU3_RESOURCES;
+
```

Comment:
Sorry for nit-picking. If we can avoid introducing non-function changes, please do so. Thanks.